projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-upload-media-v1-1-endpoint.php

```diff
@@ -192,9 +192,9 @@ function check_upload_size( $file ) {
 			$file['error'] = 'rest_upload_limited_space|' . sprintf( __( 'Not enough space to upload. %s KB needed.', 'default' ), number_format( ( $file_size - $space_left ) / KB_IN_BYTES ) );
 		}
 
-		if ( $file_size > ( KB_IN_BYTES * get_site_option( 'fileupload_maxk', 1500 ) ) ) {
+		if ( $file_size > WPCOM_MAX_UPLOAD_FILE_SIZE ) {
 			/* translators: %s: Maximum allowed file size in kilobytes. */
-			$file['error'] = 'rest_upload_file_too_big|' . sprintf( __( 'This file is too big. Files must be less than %s KB in size.', 'default' ), get_site_option( 'fileupload_maxk', 1500 ) );
+			$file['error'] = 'rest_upload_file_too_big|' . sprintf( __( 'This file is too big. Files must be less than %s KB in size.', 'default' ), WPCOM_MAX_UPLOAD_FILE_SIZE / KB_IN_BYTES );
```

Comment:
Since you touched that line, I'm afraid you'll need to fix that linting error so we can merge this.

```suggestion
			$file['error'] = 'rest_upload_file_too_big|' . sprintf( __( 'This file is too big. Files must be less than %s KB in size.', 'jetpack' ), WPCOM_MAX_UPLOAD_FILE_SIZE / KB_IN_BYTES );
```

Reply:
@jeherve  Does that mean this change has to sync back to wpcom as well?

Reply:
Yup

Reply:
ok, and I guess I have to do that manually since this PR isn't generating a diff for me.

Is there an order that this needs to happen in, or can I do that after this has merged?

Reply:
I'm afraid you're going to have to do it manually. This is one of the inconvenients of the "dotcom first" approach.